docs/user-guide/configure-overview.md

```diff
@@ -12,192 +12,166 @@ using environment variables with the same name as the property.
-        # Server port for Styx admin interface.
-        port: 9000
-
-      # Number of threads for handling incoming connections on admin interface. 0 -> availableProcessors / 2 threads will be used.
-      bossThreadsCount: 1
-
-      # Number of worker threads for admin interface
-      # Worker threads are those performing all the asynchronous I/O operation on the inbound channel.
-      # 0 -> availableProcessors / 2 threads will be used
-      workerThreadsCount: 1
-
-      tcpNoDelay: true
-      nioReuseAddress: true
-      nioKeepAlive: true
-      maxInitialLength: 4096
-      maxHeaderSize: 8192
-      maxChunkSize: 8192
-      maxContentLength: 65536
-      
-      # Whether to cache the generated JSON for the /admin/metrics and /admin/jvm pages
-      metricsCache:
-        enabled: true
-        expirationMillis: 10000
-
-    loadBalancing:
-      strategy: #Check load balancing documentation for all the possible strategies
-        factory: {class: "com.hotels.styx.client.loadbalancing.strategies.PowerOfTwoStrategy$Factory"}
- 
-
-
-      adaptive:
-        # Adaptive loadbalancer warm-up count. The count is the number of requests that has to hit the
-        # load balancer before it upswitches from round robin strategy to least response time strategy.
-        warmupCount: 100
-
-    metrics:
-      graphite:
-        enabled: true
-        # Host of the Graphite endpoint. Overrides the property from CoreFoundation
+# A string uniquely identifying the host running the application, must be different for all running instances of the application
+# the default value is suitable only for non clustered environments
+jvmRouteName: "${jvm.route:noJvmRouteSet}"
+
+proxy:
+  connectors:
+    http:
+      # Port for accessing the proxy server over HTTP.
+      port: 8080
+    https:
+      # Port for accessing the proxy server over HTTPS.
+      port: 8443
+      # Implementation of SSL functionality, can be JDK or OPENSSL.
+      sslProvider: OPENSSL
+      # Timeout for the cached SSL session objects.
+      sessionTimeoutMillis: 300000
+      # Size of the cache used for storing SSL session objects.
+      sessionCacheSize: 20000
+      # The TLS protocol versions to enable.
+      protocols:
+        - TLSv1.2
+      # The cipher suites to enable, in the order of preference.
+      cipherSuites:
+       - TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+       - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+       - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+       - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+  # Boss threads are those establishing new channels.
+  bossThreadsCount: 1
+  # Styx client worker threads are those performing all the asynchronous I/O operation to the backend origins.
+  # If set to 0, availableProcessors / 2 threads will be used
+  clientWorkerThreadsCount: 0
+  # Worker threads are those performing all the asynchronous I/O operation on the inbound channel.
+  # If set to 0, availableProcessors / 2 threads will be used
```

Comment:
I think I would still remove this "If set to 0", as IIRC this was not the case here. Then let's change the example to something different from 0!